Signal/src/call/CallAudioService.swift

```diff
@@ -169,32 +176,52 @@ struct AudioSource: Hashable {
             return
         }
 
+        // Disallow bluetooth when user has explicitly chosen the built in receiver.
+        // I'm actually not sure why this is required - it seems like we should just be able
+        // to setPreferredInput to call.audioSource.portDescription in this case,
+        // but in practice I'm seeing the call revert to the bluetooth headset.
+        // Presumably something else (in WebRTC?) is touching our shared AudioSession.
+        let options: AVAudioSessionCategoryOptions = call.audioSource?.isBuiltInEarPiece == true ? [] : [.allowBluetooth]
```

Comment:
`== true` is redundant and can be omitted?

Not in Swift - it's comparing to an optional boolean, so `== true` means it's not false **and** it's not nil.

> Will this interfere with the user connecting their bluetooth after the call is connected?

Should be fine. This only disallows bluetooth when the user has explicitly chosen "iPhone" output. If, after that, they explicitly choose a bluetooth source, then this restriction goes away. I'll try to clarify the comment.